.github/workflows/prebuilt_devcontainer.yml

```
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: 2023 SUSE LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'pre-built devcontainer'
+on:
+  pull_request:
```

Comment or remove this line please.
For now, let's only build containers for master branch, not for every PR.
```suggestion
  # pull_request:
```